tests/zfs-tests/tests/functional/cli_root/zpool_split/zpool_split_resilver2.ksh

```diff
@@ -0,0 +1,76 @@
+#!/bin/ksh -p
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2018, Nexenta Systems, Inc.  All rights reserved.
+#
+
+. $STF_SUITE/include/libtest.shlib
+
+#
+# DESCRIPTION:
+# 'zpool split' should fail if resilver in progress for a disk to be detached
+#
+# STRATEGY:
+# 1. Create a mirror pool
+# 2. Offline the second VDEV
```

Comment:
Is this the only difference (first vs second VDEV) between these two test scripts? Does it make sense to run these tests in a single test case/script instead?

Reply:
the first script checks vdev_dtl_required() and the second checks vdev_resilver_needed(). I can merge both scripts into one if that is ok.

Reply:
Thanks, that makes sense now: i somehow got confused by not seeing where `$disk` is defined in the first script:

https://github.com/zfsonlinux/zfs/blob/e9404f0d73c58c0c122f43b1f6c0f885269e369c/tests/zfs-tests/tests/functional/cli_root/zpool_split/zpool_split_resilver1.ksh#L67-L69

I think it's ok to keep the test cases separate but i wish we could make more explicit why we need to test both situations in a note/comment.